web/src/lib/domain-logic/getNavBarBusinessTitle.test.ts

```diff
@@ -141,6 +141,38 @@ describe("getNavBarBusinessTitle", () => {
         });
       });
     });
+
+    describe("Nexus DBA", () => {
+      it("shows unamed dba when the user has an unavailable name", () => {
```

Comment:
[pebble] I suggest renaming this test to describe the behavior at play here. Although it is caused by an unavailable name, yes, the actual test and logic never looks at availablity at all.

More specifically, what causes it to show unnamed DBA? It is when the user needsNexusDbaName and the nexusDbaName is empty. We should say that.

also, typo "unnamed"